pages/linux/whiptail.md

```diff
@@ -0,0 +1,27 @@
+# whiptail
+
+> Display text-based dialog boxes from shell scripts.
+
+- Display a simple message:
+
+`whiptail --title "{{title}}" --msgbox "{{message}}" {{height_in_chars}} {{width_in_chars}}`
+
+- Display a boolean choice, returning the result through the exit code:
+
+`whiptail --title "{{title}}" --yesno "{{message}}" {{height_in_chars}} {{width_in_chars}}`
+
+- Customise the text on the yes / no buttons:
+
+`whiptail --title "{{title}}" --yes-button "{{text}}" --no-button "{{text}}" --yesno "{{message}}" {{height_in_chars}} {{width_in_chars}}`
+
+- Display a text input box:
+
+`{{result_variable_name}}="$(whiptail --title "{{title}}" --inputbox "{{message}}" {{height_in_chars}} {{width_in_chars}} {{default_text}} 3>&1 1>&2 2>&3)"`
+
+- Display a password input box:
+
+`{{result_variable_name}}="$(whiptail --title "{{title}}" --passwordbox "{{message}}" {{height_in_chars}} {{width_in_chars}} 3>&1 1>&2 2>&3)"`
+
+- Display a multiple-choice menu:
+
+`OPTION=$(whiptail --title "{{title}}" --menu "{{message}}" {{height_in_chars}} {{width_in_chars}} {{menu_display_height}} "{{value_1}}" "{{display_text_1}}" "{{value_n}}" "{{display_text_n}}" ..... 3>&1 1>&2 2>&3)`
```

Comment:
@sbrl, any reason this is `OPTION` instead of `{{result_variable_name}}` like the two examples above?

Reply:
Yeah, because I forgot :P

I'll update it now.

Reply:
Oh 😋 Fair enough.